src/main/webapp/app/pool/pool-booking/pool-booking.component.ts

```diff
@@ -10,16 +10,23 @@ import { PoolBookingService } from 'app/pool/pool-booking/pool-booking.service';
 export class PoolBookingComponent implements OnInit {
     public reservationPoolId: string;
     public count: number;
+    public from: Date;
+    public to: Date;
 
     constructor(private route: ActivatedRoute, private poolBookingService: PoolBookingService) {}
 
     ngOnInit(): void {
         this.route.paramMap.subscribe(e => (this.reservationPoolId = e.get('id')));
         this.count = 1;
+        this.from = new Date();
+        this.to = new Date();
+        this.to.setHours(this.to.getHours() + 1);
```

Comment:
why +1 hour? i think it would crash when you make a reservation on 23:01 since you would like to set 24 as hours number

Reply:
it will, but it won't be tested at 23:01 so I thought it is now worth it to make such logic